src/parts/_forms.scss

```diff
@@ -56,6 +54,36 @@ textarea:focus {
   box-shadow: 0 0 0 2px $focus;
 }
 
+input[type='checkbox'],
+input[type='radio'] {
+  width: 14px;
+  height: 14px;
+  display: inline-block;
+  vertical-align: middle;
+  margin: 0;
+  margin-right: 2px;
+}
+
+input[type='radio'] {
+  border-radius: 50%;
+}
+
+input[type='checkbox']:checked,
+input[type='radio']:checked {
+  background: $button-hover;
+
+  &::before {
+    content: '•';
+    display: block;
+    transform: translateY(-50%) translateY(-0.3px) translateX(-2.7px);
+  }
+}
+
+input[type='checkbox']:checked::before {
+  content: '✔';
+  transform: translateY(-50%) translateY(0.5px) translateX(-6px);
```

Comment:
It's possible that this character also isn't exactly centered, but I think it looks okay visually. It doesn't matter if it's _actually_ mathematically centered, only that it visually looks centered.

Reply:
I agree. Checkmarks are also harder to "mathematically" center without looking off imo